app/controllers/api/base_controller.rb

```diff
@@ -1,3 +1,14 @@
 class Api::BaseController < ApplicationController
   layout nil
+
+  # Method to log login attempts
+  def log_login_attempt(user_id:, attempted_at:, successful:, ip_address:)
+    asd = 123
```

Comment:
The variable 'asd' is assigned but never used, which is unnecessary and can be removed to clean up the code.